### ISSUE TITLE
Training from scratch hyperparameters?

### ISSUE THREAD

**Issue by jremmons** [Randl/MobileNetV2-pytorch#6]
Hello, I am trying to reproduce your training-from-scratch results but I have not been successful (i.e. not able to match your reported accuracy)... I think this is probably because I am missing some critical information about the training procedure. 

I assumed that by running the command below I could repro your numbers, but I have tried this to no avail. (ran for all 400 epochs, but was about 2-3 percentage points lower than your reported numbers). 
```
python imagenet.py --dataroot "/path/to/imagenet/"
```

Could you help me answer the following questions?

1. What was the batch_size used during each epoch during training? It looks like you may have started with a batch size of 64, then up'ed it to 256 at some point? (at epoch 57?)
2. Did you train using the imagenet2012 or imagenet2017 dataset? Or did you use a different version? 
3. Did you change the momentum or the weight decay parameters from their defaults at all during training?
4. Aside from using a seed of 42, did you do anything special with initialization?

**Comment by Randl** [comment]
You can check the files:
https://github.com/Randl/MobileNetV2-pytorch/blob/master/results/mobilenet_v2_1.0_224/params.txt
https://github.com/Randl/MobileNetV2-pytorch/blob/master/results/mobilenet_v2_0.5_160/params.txt
for running parameters. The command should result in exactly the same run. The imagenet I believe is 2012 one (not sure there is any difference).

**Comment by jremmons** [comment]
Gotcha. I noticed that in [this](https://github.com/Randl/MobileNetV2-pytorch/blob/master/results/mobilenet_v2_1.0_224/params.txt) params file that you were resuming from a previous run `results/2018-05-22_21-10-55/`. Could you post the params.txt associated with that run? I just want to know the entire history of the training procedure.

**Comment by Randl** [comment]
I'll check but I'm pretty sure the parameters were same (just got to reboot pc or something)

**Comment by Randl** [comment]
Here is content of the file
```
/home/chaimb/research/MobileNet2/imagenet.py --dataroot /home/chaimb/ILSVRC/Data/CLS-LOC --seed 42 --gpus 0,1,2,3,4 -j 32 --scaling 1.0 -b 256
Namespace(batch_size=256, dataroot='/home/chaimb/ILSVRC/Data/CLS-LOC', decay=4e-05, epochs=400, evaluate=False, gamma=0.1, gpus=[0, 1, 2, 3, 4], input_size=224, learning_rate=0.01, log_interval=100, momentum=0.9, results_dir='./results', resume='', save='2018-05-22_21-10-55', scaling=1.0, schedule=[200, 300], seed=42, start_epoch=0, type='float32', workers=32)
Claimed accuracies are: 71.80% top-1, 91.00% top-5
```

Also I suppose you to check that printed `args` fit the one appearing in `params.txt`, since I could use some default parameters that changed later (though I believe I didn't).
